lib/std/build.zig

```diff
@@ -242,6 +242,13 @@ pub const Builder = struct {
             null;
     }
 
+    /// Create the given object with the builder allocator
+    pub fn create(self: Builder, obj: anytype) *@TypeOf(obj) {
+        var ptr = self.allocator.create(@TypeOf(obj)) catch unreachable;
```

Comment:
Could we make this `@panic()` on failure instead? I know that the build system is always run in a safe build mode and that `catch unreachable` is currently used everywhere in `std.build` but it's still semantically incorrect and I don't want anyone to read this standard library code and think the style is ok in general.

Reply:
Agreed, changed.